app/Http/Controllers/FrontController.php

```diff
@@ -434,6 +434,22 @@ public function saveNewFeedback(Request $request) {
         return redirect('/')->with('success', 'Thank you for the feedback! It has been received successfully.');
     }
 
+    public function newTrainerFeedback() {
+        $users = User::with('roles')->where('status', '1')->get();
+        $ins = $users->filter(function ($user) {
+            return $user->hasRole('ins');
+        });
+
+        $mtr = $users->filter(function ($user) {
+            return $user->hasRole('mtr');
+        });
+        $feedbackOptions = $ins->merge($mtr);
```

Comment:
See above. Since we are doing the same thing twice, maybe we should retrieve the feedback options from the `TrainerFeedback` model as a static function so we don't have duplicated code

Reply:
Done